samples/python/reactors/plasma.py

```diff
@@ -0,0 +1,71 @@
+"""
+Plasma Reactor
+==============
+Solve a constant pressure and constant gas/electron temperature plasma problem.
+
+This file uses a similar approach as custom.py, where the governing equations are
+implemented in Python, to solve the electron concentration in a plasma
```

Comment:
```
r = ct.Reactor()
r.energy_enabled=False
r.insert(plasma)
```
Using Reactor will have NotImplementedError thrown by PlasmaPhase::intEnergy_mole: Not implemented. How to disable the thermal model to prevent this?